### PR TITLE
Remove unwrap for safety

### PR DESCRIPTION
**Pull request by ManaSugi**
Remove all unwraps to replace them with code to return the `SeccompError`

Signed-off-by: Manabu Sugimoto <Manabu.Sugimoto@sony.com>

### CODE REVIEW

**Comment by ManaSugi** [comment]
@rusty-snake Could you review this?

File: libseccomp/src/lib.rs [hunk 1]
```diff
@@ -778,7 +773,10 @@ pub fn get_syscall_name_from_arch(arch: ScmpArch, syscall_num: i32) -> Result<St
     }
 
     let name_c: &CStr = unsafe { CStr::from_ptr(ret) };
-    let name = name_c.to_str().unwrap().to_string();
+    let name = name_c
+        .to_str()
+        .map_err(|e: std::str::Utf8Error| SeccompError::new(Common(e.to_string())))?
```
Comment (rusty-snake) [inline libseccomp/src/lib.rs:778]:
```suggestion
        .map_err(|e: std::str::Utf8Error| SeccompError::with_source(Common(e.to_string()), e))?
```

File: libseccomp/src/lib.rs [hunk 2]
```diff
@@ -791,7 +789,8 @@ pub fn get_syscall_name_from_arch(arch: ScmpArch, syscall_num: i32) -> Result<St
 /// Returns the number of the syscall, or an error if an invalid architecture is
 /// passed or a syscall with that name was not found.
 pub fn get_syscall_from_name(name: &str, arch: Option<ScmpArch>) -> Result<i32> {
-    let name_c = CString::new(name).unwrap();
+    let name_c = CString::new(name)
+        .map_err(|e: std::ffi::NulError| SeccompError::new(Common(e.to_string())))?;
```
Comment (rusty-snake) [inline libseccomp/src/lib.rs:793]:
```suggestion
        .map_err(|e: std::ffi::NulError| SeccompError::with_source(Common(e.to_string()), e))?;
```

**Review by rusty-snake — COMMENTED** [review]
LGTM but I think we can use `SeccompError::with_source` for the `map_err`s.

**Comment by ManaSugi** [comment]
@rusty-snake Thank you for the review!  I forgot the existence of `with_source` while creating it by myself.

**Pull request merged**